packages/block-library/src/media-text/edit.native.js

```diff
@@ -153,6 +158,10 @@ class MediaTextEdit extends Component {
 		};
 		const innerBlockWidth = shouldStack ? 100 : ( 100 - temporaryMediaWidth );
 		const innerBlockWidthString = `${ innerBlockWidth }%`;
+		const mediaContainerStyle = {
+			padding: isParentSelected || isAncestorSelected ? 6 : 16,
+			...( isSelected && { padding: 12 } ),
```

Comment:
Similar to the previous comment, let's try to keep the values in the stylesheet. I imagine they also come from the relations already defined through SCSS variables, and it'd be clearer to specify them that way.